src/__tests__/service-metrics.test.ts

```diff
@@ -4,6 +4,12 @@ describe('simple test of metrics', () => {
   beforeAll(async () => {
     ServiceMetrics.start()
   })
+  test('trace span', async() => {
+    const span = ServiceMetrics.start_span("doing it")
```

Comment:
```suggestion
    // We only check here if we *can call* the methods
    const span = ServiceMetrics.start_span("doing it")
```

Maybe something like that to make the intent explicit?

Reply:
sure, done.